### PR TITLE
bump core, navigation, and activities

### PR DESCRIPTION
**Pull request by wongvincent**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -4184,14 +4184,14 @@
       }
     },
     "d2l-activities": {
-      "version": "github:BrightspaceHypermediaComponents/activities#fa932e04e528af6fae09fede0026458d13053ff4",
+      "version": "github:BrightspaceHypermediaComponents/activities#4d01e08b275865daabc4279d52abf5e250260b5f",
```
Comment (wongvincent) [inline package-lock.json:4187]: @BhamelinD2L

File: package-lock.json [hunk 2]
```diff
@@ -1076,9 +1076,9 @@
       }
     },
     "@brightspace-ui/core": {
-      "version": "1.40.3",
-      "resolved": "https://registry.npmjs.org/@brightspace-ui/core/-/core-1.40.3.tgz",
-      "integrity": "sha512-p4CdvtnRw4+vRPJ62REKWT1Qx8kWD7p96c7SAGcf11weVFv/cLwJGidR+flvJD8kaypvbQ4YT8d62kqcIEQIEw==",
+      "version": "1.42.0",
```
Comment (wongvincent) [inline package-lock.json:1079]: @margaree @AllanKerr @dbatiste @omsmith

Comment (omsmith) [reply package-lock.json:1079]: :+1: Good from my end

Comment (dbatiste) [reply package-lock.json:1079]: Good on my end.

**Pull request merged**